.changeset/fast-starfishes-carry.md

```diff
@@ -0,0 +1,12 @@
+---
+"@codemod-utils/ember-cli-string": patch
+"@codemod-utils/ast-javascript": patch
+"@codemod-utils/ast-template": patch
+"@codemod-utils/blueprints": patch
+"@codemod-utils/files": patch
+"@codemod-utils/tests": patch
+"@codemod-utils/json": patch
+"@codemod-utils/cli": patch
```

Comment:
To be safe, I'll do a patch release for every `@codemod-utils/*` package (instead of just `@codemod-utils/ast-javascript` and `@codemod-utils/json`), because `typescript` was updated from `5.3.3` to `5.4.2`.